journalbeat/_meta/fields.common.yml

```diff
@@ -116,16 +116,19 @@
               required: false
```

Comment:
@kvch Could you take a look at the comments related to Journalbeat?

Reply:
Opened a PR: https://github.com/elastic/beats/pull/10542